presto-spark-launcher/pom.xml

```diff
@@ -39,7 +39,13 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-compress</artifactId>
-            <version>1.19</version>
+            <version>1.26.0</version>
```

Comment:
I see, we already have commons-compres with an updated version added in root pom. We can use the same here instead of adding a version again in this module.

https://github.com/prestodb/presto/blob/master/pom.xml#L2212
https://github.com/prestodb/presto/blob/master/pom.xml#L83